### PR TITLE
TESTPG5_Compute

### PR DESCRIPTION
**Pull request by Bhaki06**
TESTPG5_Compute

### CODE REVIEW

File: TESTPG5.NSP.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+01 #CNT2 (N2)
+01 #CNT3 (N3)
+01 #CNT4 (N3)
+END-DEFINE
+ 
+#CNT1 := 2
+#CNT2 := 2
+#CNT3 := #CNT1 * #CNT2
+#CNT4 := #CNT3
+WRITE 'COMPUTED VALUE' #CNT4
```
Comment (vsb9962011485) [inline TESTPG5.NSP.txt:13]: Please include all local variables in WRITE statement

**Review by vsb9962011485 — COMMENTED** [review]
Please fix the review commetns

File: TESTPG5.NSP.txt (hunk 1 above)
Comment (Bhaki06) [reply TESTPG5.NSP.txt:13]: Fixed ..

**Review by vsb9962011485 — APPROVED** [review]
Approved

**Pull request merged**